includes/class-wc-gateway-stripe.php

```diff
@@ -655,19 +655,32 @@ public function process_payment( $order_id, $retry = true, $force_save_source =
 	 * Displays the Stripe fee
 	 *
 	 * @since 4.0.6
-	 * @param $order_id
+	 *
+	 * @param int $order_id Reference.
```

Comment:
Reference is wrong. Should be something like "The ID of the order". Or in this case not even needed as it is self descriptive.

Reply:
ok, I based on the doc of the function process_payment (line 369)